Python/Product/PythonTools/PythonTools/Intellisense/ProjectAnalyzer.cs

```diff
@@ -1960,6 +1974,66 @@ class ExprWalker : PythonWalker {
             return new AnalysisVariable(type, location);
         }
 
+        internal static string ExpressionForDataTip(IServiceProvider serviceProvider, ITextView view, SnapshotSpan span) {
```

Comment:
Also, I'd have thought that all of our threading was either free-threaded or UI-bound - there shouldn't be a need for a new synchronization context. But @DinoV may disagree - I haven't quite caught up with the out-of-proc analyzer threading model.